docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,210 @@
 # Getting started
```

Comment:
Option 2 means there is a table of contents on the landing page, but it contains duplicated information since article titles are already on the navigation bar on the left. [According to Anne, this conflicts with the "no-stacked-headings" rule](https://github.com/canonical/pebble/pull/419#issuecomment-2144198273).

So I will go for option 1 and change the home page to singular for now.